SourceryTests/ConfigurationSpec.swift

```diff
@@ -9,26 +9,136 @@ class ConfigurationSpec: QuickSpec {
     override func spec() {
         let relativePath = Path("/some/path")
 
+        describe("Configuration") {
+            context("given invalid config file") {
+
+                func configError(_ config: [String: Any]) -> String {
+                    do {
+                        _ = try Configuration(dict: config, relativePath: relativePath)
+                        return "No error"
+                    } catch {
+                        return "\(error)"
+                    }
+                }
+
+                it("throws error on invalid file format") {
+                    do {
```

Comment:
I've tried, but for some reason it does not even call closure in `execute` and just fails. Why - is beyond me =)